5-HTML-and-CSS-projects/3-Product-landing-page/casper.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <meta name="description" content="Casper Website">
+        <title>Casper Website</title>
+        <link rel="stylesheet" type="text/css" href="style.css" media="screen">
+    </head>
+    <body>
+    <img src="casper-logo.png" class="logo"/>
+    <header>
+        <p class="top-liner">Over 35,000 5* reviews</p>
+        <h1>This is what award-wining conform feels like</h1>
+        <a href="#" class="main-button">Shop for mattresses</a>
+        <p class="footer-text">According to Casper global onsite customer reviews</p>
+    </header>
+    <div class="testimonials">
+        <h1>Our mattresses have anyone talking...</h1>
+        <div class="reviews">
+            <h2>TIME</h2>
+            <p>One of the times 25 best inventions</p>
+        </div>
+        <div class="reviews">
+            <h2>GOOD HOUSE KEEPING</h2>
+            <p>Good housekeeping 2018 lab pick</p>
+        </div>
+        <div class="reviews">
+            <h2>GOOGLE</h2>
+            <p>The perfect mattress, according to science</p>
+        </div>
+    </div>
+    <footer>
+        Copyright @ <a href="http://casper.com" target="_blank">Casper</a>
+    </footer>
+    </body>
+</html>
```

Comment:
It's good practice to leave an empty line at the end of files. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file